### ISSUE TITLE
rbeapi exits if $HOME is not set

### ISSUE THREAD

**Issue by jerearista** [arista-eosplus/rbeapi#46]
rbeapi should gracefully continue if the path expansion for '~/.eapi.conf' fails.

```
bash-3.2$ (unset HOME; irb)
irb(main):001:0> require 'rbeapi/client'
=> true
irb(main):002:0> node = Rbeapi::Client.connect_to('veos01')
ArgumentError: couldn't find HOME environment -- expanding `~'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:190:in `expand_path'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:190:in `block in autoload'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:189:in `each'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:189:in `autoload'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:171:in `initialize'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:61:in `new'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:61:in `config'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:89:in `config_for'
	from /opt/crossfader/versions/ruby/1.9.3-p448/lib/ruby/gems/1.9.1/gems/rbeapi-0.2.0/lib/rbeapi/client.rb:102:in `connect_to'
	from (irb):2
	from /opt/crossfader/versions/ruby/1.9.3-p448/bin/irb:12:in `<main>'
```

**Comment by devrobo** [comment]
Fixed with commit: 7d837d4775a6e7d3d20781033e42c413a012fc0f

**Comment by jerearista** [comment]
Release Note:
Searching for eapi.conf in $HOME would fail if $HOME was not defined.
